deploy/README.md

```diff
@@ -0,0 +1,309 @@
+
+ - `wikibase-image-data` MediaWiki image and media file uploads.
+ - `mysql-data` MediaWiki/Wikibase MariaDB raw database.
+ - `wdqs-data` Wikidata Query Service raw database.
+ - `elasticsearch-data` Elastic Search raw database.
+ - `quickstatements-data` Generated Quickstatement OAuth binding for this MediaWiki instance.
+ - `traefik-letsencrypt-data` SSL certificates.
+
+#### Backup your Data
+You can backup your data by shutting down the instance and dumping the contents of all Docker Volumes into `.tar.gz` files.
+
+```sh
+docker compose down
+
+for v in \
+    wikibase-suite-wikibase_image-data \
+    wikibase-suite_mysql-data \
+    wikibase-suite_wdqs-data \
+    wikibase-suite_elasticsearch-data \
+    wikibase-suite_quickstatements-data \
+    wikibase-suite_traefik-letsencrypt-data; do
+  docker run --rm --volume $v:/backup debian:12-slim tar cz backup > $v.tar.gz
+done
+```
+
+#### Restore from a backup
+
+To restore the volume backups, ensure your instance is shut down with `docker compose down` and populate the Docker Volumes with data from your `.tar.gz` files.
+
+```sh
+docker compose down
+
+for v in \
+    wikibase-suite-wikibase_image-data \
+    wikibase-suite_mysql-data \
+    wikibase-suite_wdqs-data \
+    wikibase-suite_elasticsearch-data \
+    wikibase-suite_quickstatements-data \
+    wikibase-suite_traefik-letsencrypt-data; do
+  docker volume rm $v 2> /dev/null
+  docker volume create $v
+  docker run -i --rm --volume $v:/backup debian:12-slim tar xz < $v.tar.gz
+done
+```
+
+### Updating
+
+WBS is versioned with [Semantic Versioning](https://semver.org/spec/v2.0.0.html). The WBS Deployment Kit and all the WBS Service Images have individual version numbers.
+
+> 💡 WBS Deployment Kit always references the latest minor and patch releases of the compatible WBS Service Images major versions.
+
+#### Minor and patch updates for WBS Service Images
+
+As WBS Deployment Kit always references latest minor and patch releases of compatible WBS Service Images TODO: rephrase, non breaking changes (including security updates) are applied automatically when recreating Docker containers. This should be always safe to do. Simply run
+
+```sh
+docker compose down 
+docker compose up --wait
+```
+
+> 💡 If you do not want to pull in new versions of WBS Service Images on container restart, just stop your containers using `docker compose stop` before restart. This will keep the current containers intact. Please note that this will not apply any security updates. It is generally recommended to remove your containers for restart with `docker compose down`.
+
+#### Minor and patch updates for WBS Deployment Kit
+
+WBS Deployment Kit major versions are tracked in dedicated branches such as `TODO:BRANCH_NAME_HERE`. Therefor, pulling from such a branch will only update minor and patch version but never trigger breaking changes.
+
+If you did not change `docker-compose.yml`, you can update simply using `git pull`.
```

Comment:
In the case of minor and patch updates I don't think we make changes to `deploy/docker-compose.yml`

Reply:
I could think of cases. Maybe updating traefik from 2.5 to 2.6 could be one of them.

Reply:
I mean, technically we could even add features such as an reverse proxy, except that we did that already ^^

Reply:
Fair, I guess I was assuming we'd move up a major version for deploy kit at such changes, but technically yes we could add non-breaking configuration or additional services and consider them non-breaking/minor
updates...